src/PythonExtensionGen.cpp

```diff
@@ -111,7 +111,7 @@ void PythonExtensionGen::convert_buffer(const string &name, const LoweredArgumen
     dest << /*name*/ "\"" << name << "\"";
     dest << ") < 0) {\n";
     release_buffers("        ");
-    dest << "        return NULL;\n";
```

Comment:
These quoted ones also worth considering.

Reply:
Since our C++ backend already requires C++11, I think it's fair to assume that the Python extensions should, too. But can revert these if you like.

Reply:
Seems reasonable. Just making sure we're doing this intentionally.